packages/amplify-cli/src/utils/set-ops.ts

```diff
@@ -0,0 +1,47 @@
+export function twoStringSetsAreEqual(a: Set<string>, b: Set<string>): boolean {
+  if (a.size !== b.size) {
+    return false;
+  }
+
+  for (const item of a) {
+    if (!b.has(item)) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+export function twoStringSetsAreDisjoint(a: Set<string>, b: Set<string>): boolean {
+  if (a.size > b.size) {
+    var temp = a;
+    a = b;
+    b = temp;
+  }
+
+  for (const item of a) {
+    if (b.has(item)) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+export function findIntersections(a: Set<string>, b: Set<string>): Set<string> {
```

Comment:
Please use the "loadash" lib instead o implementing all these functions yourself

Reply:
I googled around, seems like lodash does not have set operations neither
https://github.com/lodash/lodash/issues/2981